src/Component/src/Metadata/Api/Delete.php

```diff
@@ -24,6 +24,7 @@ final class Delete extends HttpOperation implements DeleteOperationInterface, Ap
 {
     public function __construct(
         ?string $path = null,
+        ?string $routeName = null,
```

Comment:
I'm considering releasing this in a patch because it could potentially be a BC break and could affect the end application 🤔

Reply:
Do you want me to move it at the end of the constructor? Normally, lot of people is using name parameter but you're right. I don't remember if theses classes are marked as ecperimental.

Reply:
They're all experimental, and given the number of arguments, it's highly doubtful that anyone would just use them in order 🤷